client/components/RoomList.jsx

```diff
@@ -14,31 +23,67 @@ const RoomList = React.createClass({
 
   componentDidMount() {
     $('.ui.accordion').accordion();
+    Meteor.call("get.room.counts", (error, response) => {
+      this.setState(response);
```

Comment:
What does this actually do? Very hard to know what you are querying and what you are changing.